### PR TITLE
docs: update README with latest setup

### LINKED ISSUE #275: Include .env creation on fuel-explorer docs. (failing after pnpm install)

### PR DESCRIPTION
**Pull request by helciofranco**
- Add missing instructions to run e2e tests.
- Remove duplicated packages between `portal` and `explorer`.
- Add scripts to run `storybooks`.
- Replace `date-fns` with `dayjs` in portal (in order to keep only one date library in the project).

Closes #275

### CODE REVIEW

**Review by arthurgeron — APPROVED** [review]

**Comment by HoTandy** [comment]
@helciofranco I'm getting a `404` when clicking on `View deployment`. Has something gone wrong since you deployed?

**Comment by helciofranco** [comment]
@HoTandy I've just started a new deploy, there was an issue with the infra setup (not related with my changes)
this should be all set in 10 minutes.

**Review by HoTandy — APPROVED** [review]
I think this is looking better now, less confusion and now referencing the correct .env file

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -39,7 +38,9 @@
     "node:stop": "make -C ./docker stop",
     "prepare": "husky install",
     "start": "pnpm turbo:run start",
-    "storybook:portal": "pnpm turbo:run storybook --filter=app-portal",
+    "storybook": "pnpm turbo:run dev:storybook",
+    "storybook:explorer": "pnpm turbo:run dev:storybook --filter=app-explorer",
+    "storybook:ui": "pnpm turbo:run dev:storybook --filter=@fuels/ui",
```
[the remark this answers is not in the document]
Comment (helciofranco) [reply package.json]: I think so!

https://github.com/FuelLabs/fuel-explorer/pull/322/commits/a8df239de93b2553f618b64525f6e0280f80b68c

**Review by LuizAsFight — DISMISSED** [review]

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**